### PR TITLE
add new friction model spatialVoellmy [com1]

### PR DESCRIPTION
**Pull request by awirb**
This allows the loading of mu/xi file via raster

### CODE REVIEW

**Review by fso42 — CHANGES_REQUESTED** [review]
- [x] Add spatialVoellmy to the description of friction models https://docs.avaframe.org/en/latest/theoryCom1DFA.html#friction-model

File: avaframe/com1DFA/checkCfg.py [hunk 1]
```diff
@@ -65,7 +65,7 @@ def checkCellSizeKernelRadius(cfg):
     return cfg
 
 
-def checkCfgFrictionModel(cfg, relVolume=''):
+def checkCfgFrictionModel(cfg, inputSimFiles, relVolume=''):
```
Comment (fso42) [inline avaframe/com1DFA/checkCfg.py:68]: Please add inputSimFiles to docstring

Comment (fso42) [inline avaframe/com1DFA/checkCfg.py:68]: Update all checkCfgFrictionModel calls with new parameter (there are still some in test_checkCfg)

File: avaframe/com1DFA/checkCfg.py [hunk 2]
```diff
@@ -140,4 +140,17 @@ def checkCfgFrictionModel(cfg, relVolume=''):
             else:
                 log.info('Friction model parameter used: %s with value %s' % (frictP, cfg['GENERAL'][frictP]))
 
+    # if spatialVoellmy check if mu and xi file are provided
+    if frictModel.lower() == 'spatialvoellmy':
+        if inputSimFiles['muFile'] == None:
```
Comment (fso42) [inline avaframe/com1DFA/checkCfg.py:145]: please use 'is' instead of ==

File: avaframe/com1DFA/checkCfg.py [hunk 3]
```diff
@@ -140,4 +140,17 @@ def checkCfgFrictionModel(cfg, relVolume=''):
...
+            message = 'No file for mu found'
+            log.error(message)
+            raise FileNotFoundError(message)
+        elif inputSimFiles['xiFile'] == None:
```
Comment (fso42) [inline avaframe/com1DFA/checkCfg.py:149]: samer here

File: avaframe/com1DFA/deriveParameterSet.py [hunk 4]
```diff
@@ -788,6 +788,8 @@ def checkRasterMeshSize(cfgSim, rasterFile, typeIndicator='DEM', onlySearch=Fals
     # if cell size of raster is different from desired meshCellSize - look for remeshed raster or remesh
     if np.abs(meshCellSize - headerRaster['cellsize']) > meshCellSizeThreshold:
         pathToRaster = geoTrans.remeshRaster(rasterFile, cfgSim, onlySearch=onlySearch)
+        if typeIndicator == 'FRIC':
```
Comment (fso42) [inline avaframe/com1DFA/deriveParameterSet.py:791]: Why is this here, but not in the else?

And why is the pathToRaster relative in the case of FRIC?

File: docs/theoryCom1DFA.rst [hunk 5]
```diff
@@ -483,6 +483,11 @@ in order to take advantage of both. This leads to the following friction law:
 
 where :math:`\xi` is the turbulent friction term. This model is described as Voellmy-Fluid :cite:`Sa2004,Sa2007`.
 
+
+There is also the option to use spatially variable values for the friction parameters :math:`\mu =f(x, y)` and :math:`\xi =f(x, y)`.
```
Comment (fso42) [inline docs/theoryCom1DFA.rst:487]: It is also possible to use ...

**Review by fso42 — CHANGES_REQUESTED** [review]

File: avaframe/com1DFA/deriveParameterSet.py (hunk 4 above)
Comment (awirb) [reply avaframe/com1DFA/deriveParameterSet.py:791]: changed it to not be the relative one

File: docs/theoryCom1DFA.rst (hunk 5 above)
Comment (awirb) [reply docs/theoryCom1DFA.rst:487]: updated

**Review by fso42 — APPROVED** [review]
